keyboards/hineybush/sm68/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "sm68",
+    "url": "",
+    "maintainer": "hineybush",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label":"k00", "x":0, "y":0},
+                {"label":"k01", "x":1, "y":0},
+                {"label":"k02", "x":2, "y":0},
+                {"label":"k10", "x":0, "y":1, "w":1.5},
+                {"label":"k12", "x":1.5, "y":1, "w":1.5}
+            ]
+        }
+    }
+}
```

Comment:
I guessed that this would have a Magicforce 68 footprint.

[info.json](https://github.com/qmk/qmk_firmware/files/3814289/info.json.txt)

Reply:
Yeah, that/VA68m

Reply:
That attached `info.json` file above should suit you then.